src/esp/gfx/LightSetup.cpp

```diff
@@ -28,11 +28,11 @@ Magnum::Vector4 getLightPositionRelativeToCamera(
                         "w == 1 for a point light");
 
   switch (light.model) {
-    case LightPositionModel::OBJECT:
+    case LightPositionModel::Object:
```

Comment:
Why the change in style?

Reply:
This was a refactor in other enums through Habitat-Sim we decided [upon a few months ago](https://github.com/facebookresearch/habitat-sim/pull/1009).  Guess this enum got missed.